docs/migration/01_migration_guide_to_v0.5.ipynb

```diff
@@ -438,7 +414,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`GroverOptimizer` supports both `QuantumInstance` and `BaseSampler`. But users must specify one of them."
+    "`GroverOptimizer` supports `BaseSampler` instead of `QuantumInstance`."
```

Comment:
I imagine this is needed because we still run this. I think what we did in Nature (or Max more correctly did) was to export it as a .rst so the content could stay the same, and still applied to migrating to the stated version, but it was no longer run.

Reply:
I got it. Since it's a migration guide for 0.5, I should not change it. I replace it with rst file.